src/neo/Consensus/ConsensusContext.cs

```diff
@@ -377,10 +377,7 @@ public void Reset(byte viewNumber)
                 {
                     PrevHash = Snapshot.CurrentBlockHash,
                     Index = Snapshot.Height + 1,
-                    NextConsensus = Blockchain.GetConsensusAddress(
-                        NativeContract.NEO.ShouldRefreshCommittee(Snapshot.Height + 1) ?
-                        NativeContract.NEO.ComputeNextBlockValidators(Snapshot) :
-                        NativeContract.NEO.GetNextBlockValidators(Snapshot))
+                    NextConsensus = Blockchain.GetConsensusAddress(NativeContract.NEO.GetNextNextBlockValidators(Snapshot))
```

Comment:
I am not sure if it's complicate the wheel, maybe we can create a method for `GetNext Consensus` and call it here.

Reply:
fixed